### DISCUSSION TITLE
New settings in offline mode are not clear

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#268] [Q&A] [answered]
It is not obvious what the settings in the new offline mode mean. E.g. does the switch need to be set to ON or OFF so no updates are downloaded?
Same for telemetry data.

And in general switches should not be coloured in OFF position.

**Comment by mikegapinski** [comment, marked as answer]
makes sense, I won't color them when off. https://github.com/tesla-android/flutter-app/issues/22
 
Offline mode, telemetry and updates are enabled by default and can be disabled(the defaults match how it worked in previous versions)

**Comment by RetoHB** [comment]
I am sorry to get back again. But I think the following sentence is missleading: "With this option enabled they won't immediately start downloading". Enabled means on. But from the above I understand this option must be switched off so no updates will be downloaded?
